Code/Source/Entity/Terrain/TerrainData.cpp

```diff
@@ -12,99 +12,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include <AzCore/Serialization/EditContext.h>
-#include <AzCore/Serialization/SerializeContext.h>
 #include <AzFramework/Physics/HeightfieldProviderBus.h>
-#include <Entity/TerrainEntityManagerSystemComponent.h>
-#include <Utilities/RGLUtils.h>
```

Comment:
Please do not remove this include - I am not able to build my project without it.